packages/ui/lib/components/TrackTable/Cells/TitleCell.tsx

```diff
@@ -71,7 +72,7 @@ const TitleCell: React.FC<CellProps<Track> & TrackTableExtraProps<Track>> = ({
         withAddToQueue={Boolean(onAddToQueue)}
         withAddToFavorites={Boolean(onAddToFavorites)}
         withAddToPlaylist={Boolean(onAddToFavorites)}
-        withAddToDownloads={Boolean(onAddToDownloads)}
```

Comment:
There is no need for an additional flag here. The presence of this button should be controled only by the presence of the `onAddToDownloads` callback. If it doesn't work, we should check why, and fix the cause.

Reply:
It looks like there's no way to pass props to the track from library view( packages\app\app\components\LibraryView) to the track/track pop up.
I think the onAddToDownloads is set from the tracktablecontainer (packages\app\app\containers\TrackTableContainer\index.tsx) independent of whatever props you pass. Maybe there's a way to modify the tracks that are passed to this container so that it doesn't render the download button? I'm new so I'm not certain about any of this

Reply:
@Josef-Kretz This is my first contribution to this repo, and I'm not familiar with the code either. I've changed it, now you should pass prop to TrackTableContainer - then if the prop is set to true, the handler is passed (or not). I think it is correct, I will wait for @nukeop  to approve it.